CONTRIBUTING.md

```diff
@@ -1,12 +1,15 @@
 # Contributing
 
-We welcome contributions to our code, and will always endeavor to respond in a quick and helpful manner to Pull Requests.
+We welcome contributions to our code, and will always endeavor to respond in a quick and helpful manner to Pull Requests (PRs).
 
 1. **Please sign one of the contributor license agreements below** (simple and pain-free).
 1. Fork the repo, develop and test your code changes, add docs.
+1. A PR should be a set of changes that addresses one issue in the issue tracker. Please file one PR per issue, and address one issue per PR. If you want to make a change that doesn't have a corresponding issue in the issue tracker, please [create a new issue][createissue] first.
```

Comment:
suggest removing "PR" in favor of actually typing it.   Looks more professional and will be easier to understand by someone not familiar with GitHub.   

_please [submit a new issue][createissue] first._  vs  _please [create a new issue][createissue] first._

I think you normally would say please submit an issue request.    Semantics i guess.